cli/lib/appdata.ts

```diff
@@ -111,6 +112,40 @@ export async function saveAppdata( userData: UserData ): Promise< void > {
 	}
 }
 
+const LOCKFILE_PATH = path.join( getAppdataDirectory(), 'appdata-v1.lock' );
```

Comment:
NIT but I would go with `appdata-v1.json.lock`, IMO it looks more clear that it lock specifically for `appdata-v1.json`.
```suggestion
const LOCKFILE_PATH = path.join( getAppdataDirectory(), 'appdata-v1.json.lock' );
```